readme.md

```diff
@@ -215,6 +215,7 @@ Integration tests are run under the `it` profile with the Failsafe plugin using
 - Fix for JENKINS-63071, we are now setting the repository browser also for multibranch projects. Existing projects will
   need to be opened and saved again to fix this issue. New projects will work without any additional steps.
 - JENKINS-60342 added support for Pull Request triggers
+- Fix issue JENKINS-65541
```

Comment:
maybe elaborate a little bit more, it is a bit annoying to have to go and find the issue to work out if this is valuable for me or now.  Something like this maybe
```suggestion
- Fix issue JENKINS-65541 use the folder credentials if they exist for multbranch scans
```